components/Header.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export function Header() {
+  return (
+    <View style={styles.header}>
+      <Text style={styles.title}>{'My Todos'}</Text>
```

Comment:
You dont need {''} around the title
`<Text style={styles.title}>My Todos</Text>`

Reply:
I don't know why I did that.